Rocket.Chat/Extensions/UIBarButtonItem+Badge.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  UIBarButtonItem+Badge.swift
+//  Rocket.Chat
+//
+//  Created by Lucas Woo on 4/26/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+import ObjectiveC
+
+extension UIBarButtonItem {
+
+    struct BadgeExtension {
+        static var badge = "badge"
+        static var badgeEdgeInsets = "badgeEdgeInsets"
+        static var badgePadding = "badgePadding"
+        static var badgeMinSize = "badgeMinSize"
+        static var badgeLabel = "badgeLabel"
```

Comment:
their addresses are used as a unique key for storing additional properties since extension doesn't allow new stored properties.